src/main/java/org/sagebionetworks/web/client/AppLoadingView.java

```diff
@@ -21,7 +21,7 @@ public AppLoadingView() {
      * DO NOT ADD THIS TO THE MAIN IMAGE BUNGLE! This is shown while the image bundle is being download
      * to the browser so putting it in the bundle defeats the whole purpose of this image.
      */
-    final Image ajaxImage = new Image("images/main-page-load.gif");
+    final Image ajaxImage = new Image("/images/main-page-load.gif");
```

Comment:
Not sure when this is used, but this was the only other reference that I could find in the codebase with the broken pattern